aiida/manage/configuration/schema/config-v5.schema.json

```
@@ -323,14 +323,21 @@
                     "description": "RabbitMQ arguments that will be encoded as query parameters",
                     "type": "object",
                     "default": {
-                        "heartbeat": 600
+                        "heartbeat": 600,
+                        "consumer_timeout": false
```

```suggestion
                        "consumer_timeout": 1
```

lets try setting to really low, if that breaks rabbitmq 3.8